### DISCUSSION TITLE
xinference部署的多模态模型，无法出现图片输入

### DISCUSSION THREAD

**Discussion by sunhaha123** [langgenius/dify#4153] [General]
![image](https://github.com/langgenius/dify/assets/27773261/430a80ef-2062-420f-9228-5171622d4fde)
![image](https://github.com/langgenius/dify/assets/27773261/9b983aa4-f8fc-4df6-885b-d93ea9089dbf)

**Comment by ricky977** [comment]
Have you solved the embedding problem of the qwen vl model
